R/arcadia.R

```diff
@@ -85,6 +86,16 @@ SEMIBOLD_FONT <- "Suisse Int'l Semi Bold"
 MEDIUM_FONT <- "Suisse Int'l Medium"
 MONO_FONT <- "Suisse Int'l Mono"
 
+# Use OS default sans fallback if Suisse is not installed
+FONT <- NULL
```

Comment:
what does `FONT <- NULL` protect against here? will it auto default to sans or something?

Reply:
Copilot mistake I forgot to remove 😅